### PR TITLE
Maven stuff + Updated dependencies

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,138 +1,288 @@
...
-    		<version>2.0.26</version>
-    		<scope>system</scope>
-    		<systemPath>${project.basedir}/lib/xAuth.jar</systemPath>
-    	</dependency>
+			<groupId>javax.mail</groupId>
+			<artifactId>mail</artifactId>
+			<version>1.5.0-b01</version>
+		</dependency>
+		
+		<!-- Attribute, THIS DOESN?T WORK BECAUSE XEPHI USED A MODIFIED VERSION
```
Comment (Xephi) [inline pom.xml]: (a)

**Comment by Xephi** [comment]
I wait for travis build and i will check it locally before merging, but really big thanks for theses changes ^^

**Comment by Xephi** [comment]
it appears Travis Build failed due to attribute api :')

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:27 min
[INFO] Finished at: 2015-06-17T17:20:53+00:00
[INFO] Final Memory: 19M/210M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project AuthMe: Compilation failure: Compilation failure:
[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[8,24] cannot find symbol
[ERROR] symbol:   class Nonnull
[ERROR] location: package javax.annotation
[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[9,24] cannot find symbol
[ERROR] symbol:   class Nullable
[ERROR] location: package javax.annotation
[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[142,35] cannot find symbol
[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[151,39] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[160,30] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[169,30] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[8,24] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: package javax.annotation

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[9,24] cannot find symbol

[ERROR] symbol:   class Nullable

[ERROR] location: package javax.annotation

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[142,35] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[151,39] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[160,30] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[169,30] cannot find symbol

[ERROR] symbol:   class Nonnull

[ERROR] location: class com.comphenix.attribute.Attributes.Attribute

[ERROR] /home/travis/build/Xephi/AuthMeReloaded/src/main/java/com/comphenix/attribute/Attributes.java:[362,45] cannot find symbol

[ERROR] symbol: class Nullable

[ERROR] -> [Help 1]
```

**Comment by sgdc3** [comment]
Ups! I didn't check compiling under linux ;)

**Comment by sgdc3** [comment]
Also i didn't check code style...

**Comment by sgdc3** [comment]
I think it's because initially i've deleted your attribute version and later i readded it without checking anything.

**Comment by Xephi** [comment]
nah it's due to oraclejdk7 i think ^^

**Comment by sgdc3** [comment]
It isn't a problem with the compiler.... It compiles correctly on Windows 10 64 bit with JDK 1.8u45

**Comment by Xephi** [comment]
Fixed : https://github.com/Xephi/AuthMeReloaded/commit/6d84cdabe78a8f89efe0e842466ef26f2f52be51
Thanks for your hard work !

**Comment by sgdc3** [comment]
Thank you for this awesome plugin! :D

**Comment by Xephi** [comment]
Maybe you can help me, i got that kind of error on jenkins, idk why : 
```
[ERROR] Failed to execute goal on project AuthMe: Could not resolve dependencies for project fr.xephi.authme.AuthMe:AuthMe:jar:5.0-SNAPSHOT: Failed to collect dependencies at com.onarandombox.multiversecore:Multiverse-Core:jar:2.5 -> me.main__.util:SerializationConfig:jar:1.7 -> org.bukkit:bukkit:jar:1.0.1-R2-SNAPSHOT: Failed to read artifact descriptor for org.bukkit:bukkit:jar:1.0.1-R2-SNAPSHOT: Could not transfer artifact org.bukkit:bukkit:pom:1.0.1-R2-SNAPSHOT from/to ess-repo (https://ci.drtshock.net/plugin/repository/everything/): Received fatal alert: internal_error -> [Help 1]
```

**Comment by sgdc3** [comment]
It can't download Multiverse Core :/ Try to delete .m2 folder.

**Comment by sgdc3** [comment]
It could also be an issue with multiverse pom, it's very outdated...

**Comment by Xephi** [comment]
Yeah, i've seen that :p I've try to delete the .m2 folder, but nothing change, i will looking in a bit more x) Maybe re-add the Multiverse .jar into libs folder will do the trick, i will try :/

**Comment by sgdc3** [comment]
I hate this use of system libraries... I think it's better to find another solution.

**Comment by sgdc3** [comment]
I found the solution:

        <dependency>
            <groupId>com.onarandombox.multiverseinventories</groupId>
            <artifactId>Multiverse-Inventories</artifactId>
            <version>2.5</version>
            <type>jar</type>
            <scope>compile</scope>
        </dependency>

**Comment by sgdc3** [comment]
Try this instead of a system library.

**Comment by Xephi** [comment]
that's why i got systems libraries :$ i always got that kind of problem x)

**Comment by sgdc3** [comment]
Also i see in jenkins log that https://ci.drtshock.net/plugin/repository/everything/ report is causing conflicts problem, change it to https://ci.drtshock.net/plugin/repository/everything/net/ess3/

**Pull request merged**